ubuntudesign/vanilla-framework
# Restructure Jekyll work for GitHub pages

This moves jekyll files to the `/docs` folder, as that's [the only subfolder GitHub will accept](https://help.github.com/articles/configuring-a-publishing-source-for-github-pages/#publishing-your-github-pages-site-from-a-docs-folder-on-your-master-branch).

It also restructures the Jekyll files slightly, and moves the actual documentation to `/docs/src`

Once this is merged, <https://ubuntudesign.github.io/vanilla-framework> should look truly beautiful.

QA
---

You can see an example of how this works on GitHub pages at <https://nottrobin.github.io/vanilla-framework/>.

To try it locally, pull down the code and run:

``` bash
bundle install
npm install
gulp jekyll
```

Then browse to <http://127.0.0.1:4000/vanilla-framework/>.
